templates/go.mod.tpl

```diff
@@ -10,8 +10,8 @@ require (
 	github.com/onosproject/onos-api/go v0.9.14
 	github.com/onosproject/onos-lib-go v0.8.13
 	github.com/openconfig/gnmi v0.0.0-20210914185457-51254b657b7d
```

Comment:
I think you could change this to `v0.0.0-20220617175856-41246b1b3507` also. The compiler corrects it when it does `go mod tidy` but it's better to keep it current here too

Reply:
Sorry I didn't notice this comment, I'll fix it in the next patch.